### PR TITLE
Update phpcs script and dependent libs

### PR DESCRIPTION
**Pull request by joshcanhelp**
### Changes

**This PR does not change plugin functionality.**

- Rename `phpcs-ruleset.xml` → `phpcs.xml.dist` (automatically picked up by PHPCS)
- Add local PHPCS config files to `.gitignore`
- Update coding standard Composer packages
- Adjust Composer scripts

**Note:** I had to run the auto-formatter command to pass CI. I marked all the modified files with a comment if the changes were whitespace only. Non-whitespace changes in [this commit](https://github.com/auth0/wp-auth0/commit/ab381efba151886af8e477cbfd1de20aaafb776d).

### References

- [Configuration feedback](https://github.com/auth0/auth0-PHP/pull/284#issuecomment-418809388) from @jrfnl

### CODE REVIEW

File: lib/WP_Auth0_Api_Client.php [hunk 1]
```diff
@@ -114,7 +114,8 @@ public static function get_token( $domain, $client_id, $client_secret, $grantTyp
 		$headers['content-type'] = 'application/x-www-form-urlencoded';
```
Comment (joshcanhelp) [inline lib/WP_Auth0_Api_Client.php]: Whitespace-only

File: lib/WP_Auth0_Api_Operations.php [hunk 2]
```diff
@@ -116,7 +116,10 @@ public function create_wordpress_connection( $app_token, $migration_enabled, $pa
 				$enabled_clients = array_diff( $db_connection->enabled_clients, array( $client_id ) );
```
Comment (joshcanhelp) [inline lib/WP_Auth0_Api_Operations.php]: Whitespace-only

File: lib/WP_Auth0_DBManager.php [hunk 3]
```diff
@@ -171,7 +171,8 @@ public function install_db( $version_to_install = null, $app_token = '' ) {
 				}
```
Comment (joshcanhelp) [inline lib/WP_Auth0_DBManager.php]: Whitespace-only

File: lib/WP_Auth0_EditProfile.php [hunk 4]
```diff
@@ -158,7 +158,10 @@ public function override_email_update() {
 			$requires_verified_email = $this->a0_options->get( 'requires_verified_email' );
```
Comment (joshcanhelp) [inline lib/WP_Auth0_EditProfile.php]: Whitespace-only

File: lib/WP_Auth0_LoginManager.php [hunk 5]
```diff
@@ -221,7 +221,11 @@ public function redirect_login() {
```
Comment (joshcanhelp) [inline lib/WP_Auth0_LoginManager.php]: Whitespace-only

File: lib/admin/WP_Auth0_Admin.php [hunk 6]
```diff
@@ -30,7 +30,9 @@ public function admin_enqueue() {
 		wp_register_script( 'wpa0_bootstrap', WPA0_PLUGIN_BS_URL . 'js/bootstrap.min.js', array( 'jquery' ), '3.3.6' );
```
Comment (joshcanhelp) [inline lib/admin/WP_Auth0_Admin.php]: Whitespace-only

File: lib/admin/WP_Auth0_Admin_Advanced.php [hunk 7]
```diff
@@ -720,7 +720,8 @@ public function migration_ws_validation( $old_options, $input ) {
 					array(
```
Comment (joshcanhelp) [inline lib/admin/WP_Auth0_Admin_Advanced.php]: Whitespace-only

File: lib/initial-setup/WP_Auth0_InitialSetup.php [hunk 8]
```diff
@@ -144,7 +144,8 @@ public function cant_create_client_grant_message() {
 					<?php
```
Comment (joshcanhelp) [inline lib/initial-setup/WP_Auth0_InitialSetup.php]: Whitespace-only

File: lib/initial-setup/WP_Auth0_InitialSetup_Consent.php [hunk 9]
```diff
@@ -73,7 +73,11 @@ public function exchange_code() {
 		$client_id = get_bloginfo( 'url' );
```
Comment (joshcanhelp) [inline lib/initial-setup/WP_Auth0_InitialSetup_Consent.php]: Whitespace-only

File: lib/initial-setup/WP_Auth0_InitialSetup_Migration.php [hunk 10]
```diff
@@ -27,7 +27,8 @@ public function render( $step ) {
 			array(
```
Comment (joshcanhelp) [inline lib/initial-setup/WP_Auth0_InitialSetup_Migration.php]: Whitespace-only

File: lib/twitter-api-php/TwitterAPIExchange.php [hunk 11]
```diff
@@ -320,7 +320,8 @@ private function buildAuthorizationHeader( array $oauth ) {
```
Comment (joshcanhelp) [inline lib/twitter-api-php/TwitterAPIExchange.php]: Whitespace-only

File: tests/testConstantSettings.php [hunk 12]
```diff
@@ -67,9 +67,12 @@ public function testConstantPrefixFilter() {
 		$this->assertEquals( self::DEFAULT_CONSTANT_PREFIX . 'DOMAIN', $opts->get_constant_name( $opt_name ) );
```
Comment (joshcanhelp) [inline tests/testConstantSettings.php]: Whitespace-only

File: tests/testEmailVerification.php [hunk 13]
```diff
@@ -76,9 +76,11 @@ public function testHooks() {
 	 */
```
Comment (joshcanhelp) [inline tests/testEmailVerification.php]: Whitespace-only

File: tests/testLoginManager.php [hunk 14]
```diff
@@ -24,10 +24,13 @@ public function testUserinfoScope() {
 		$this->assertEquals( 'openid email profile', $scope );
```
Comment (joshcanhelp) [inline tests/testLoginManager.php]: Whitespace-only

File: tests/testUserRepoCreate.php [hunk 15]
```diff
@@ -140,9 +140,11 @@ public function testShouldCreateUserFilterRejected() {
 		update_option( 'users_can_register', 1 );
```
Comment (joshcanhelp) [inline tests/testUserRepoCreate.php]: Whitespace-only

File: tests/testWPAuth0Options.php [hunk 16]
```diff
@@ -68,7 +68,9 @@ public function testThatFiltersOverrideValues() {
 			// phpcs:ignore
```
Comment (joshcanhelp) [inline tests/testWPAuth0Options.php]: Whitespace-only

File: tests/traits/hookHelpers.php [hunk 17]
```diff
@@ -24,13 +24,15 @@ public function get_hook( $hook = '' ) {
```
Comment (joshcanhelp) [inline tests/traits/hookHelpers.php]: Whitespace-only

**Comment by joshcanhelp** [comment]
@jrfnl - Thank you so much! I probably should have waited to see if you commented 😄 I'll get another PR through for this.

**Comment by jrfnl** [comment]
@joshcanhelp No worries, were only two minor nitpicks anyway.

**Pull request merged**